pkg/api/validation/validation.go

```diff
@@ -2844,10 +2799,16 @@ func ValidateNodeUpdate(node, oldNode *api.Node) field.ErrorList {
 	// Clear status
 	oldNode.Status = node.Status
 
+	// update taints
```

Comment:
How about we say that adding/modifying tolerations is an "alpha feature" that is disabled by default, so that people who are OK with it can enable it? I realize that makes this PR more complicated than either "allow it" or "don't allow it" but I am worried about the usability implications of not having any way for people to handle the scenarios I described.